styleguide/source/assets/scss/02-molecules/_sign-in-dropdown-dr-finder.scss

```diff
@@ -0,0 +1,103 @@
+.dr-finder__sign-in-dropdown {
```

Comment:
@payekumi Do we need all this?

Reply:
This sign-in-dropdown-dr-finder.scss file has also been removed